rust/sysdb/src/sysdb.rs

```diff
@@ -143,18 +160,18 @@ impl SysDb {
     ) -> Result<Vec<Collection>, GetCollectionsError> {
         match self {
             SysDb::Grpc(grpc) => {
-                // TODO(c-gamble): Move this to config
-                let max_get_collections_limit = 100u32;
                 match options.limit {
                     Some(limit) => {
-                        if limit > max_get_collections_limit {
-                            return Err(GetCollectionsError::MaximumLimitExceeded(
-                                limit,
-                                max_get_collections_limit,
+                        if limit > MAX_LIMIT_VALUE {
```

Comment:
Did we converge on enforcing this for local also? JC, what the eventual decision was